contribute/backend/database.md

```diff
@@ -73,21 +75,21 @@ For transactions, use the `WithTransactionalDbSession` method instead.
 
 ## Migrations
 
-As Grafana evolves, it becomes necessary to create _schema migrations_ for one or more database tables.
+As your use of Grafana evolves, it may become necessary to create _schema migrations_ for one or more database tables.
 
 To see all the types of migrations you can add, refer to [migrations.go](/pkg/services/sqlstore/migrator/migrations.go).
 
 Before you add a migration, make sure that you:
 
-- Never change a migration that has been committed and pushed to main.
+- Never change a migration that has been committed and pushed to `main`.
 - Always add new migrations, to change or undo previous migrations.
 
 Add a migration using one of the following methods:
 
 - Add migrations in the `migrations` package.
 - Implement the `DatabaseMigrator` for the service.
 
-**Important:** If there are previous migrations for a service, use that method. By adding migrations using both methods, you risk running migrations in the wrong order.
+**Important:** If there are previous migrations for a service, use that method. Don't add migrations using both methods or you risk running migrations in the wrong order.
```

Comment:
could this be a note? maybe consider a shortcode for this, too

Reply:
fixed